packages/sysutils/busybox/scripts/create-edid-cpio

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2021-present Team LibreELEC (https://libreelec.tv)
+
+EDID_DIR="/storage/.config/firmware/edid"
```

Comment:
I don't really understand, so you need to have an edid already to make an edid cpio?

Why can't you get the edid from the drm subsystem via a valid connector?

We already have the instructions here -> https://libreelec.wiki/configuration/edid